install/kubernetes/cilium/charts/agent/templates/svc.yaml

```
@@ -17,3 +17,43 @@ spec:
   selector:
     k8s-app: cilium
 {{- end }}
+{{- if .Values.global.hubble.hubbleUi.enabled }}
+---
+kind: Service
+apiVersion: v1
+metadata:
+  name: hubble-grpc
+  namespace: {{ .Release.Namespace }}
+  labels:
+    k8s-app: hubble
+spec:
+  type: ClusterIP
+  clusterIP: None
+  selector:
+    k8s-app: hubble
+  ports:
+  - targetPort: 50051
+    protocol: TCP
+    port: 50051
+{{- end }}
+{{- if and .Values.global.hubble.metrics.enabled (.Values.global.hubble.metrics.serviceMonitor.enabled) }}
+---
+kind: Service
+apiVersion: v1
+metadata:
+  name: hubble-metrics
+  namespace: {{ .Release.Namespace }}
+  labels:
+    k8s-app: hubble-metrics
```

```suggestion
    k8s-app: cilium
```

Hubble is now running inside Cilium. Therefore this needs to target the cilium pod.

---

Fixed